service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ReviewCMOControllerAboutToSubmitTest.java

```diff
@@ -44,20 +48,32 @@
     @MockBean
     private DocumentConversionService documentConversionService;
 
+    @MockBean
+    private FeatureToggleService featureToggleService;
+
+    private CaseManagementOrder cmo = buildCMO();
+    private DocumentReference convertedDocument;
+    private DocumentReference sealedDocument;
+
     ReviewCMOControllerAboutToSubmitTest() {
         super("review-cmo");
     }
 
+    @BeforeEach()
+    void setup() throws Exception {
+        convertedDocument = testDocumentReference();
+        sealedDocument = testDocumentReference();
+    }
+
```

Comment:
I think you mentioned about reverting it? not the case? Or you happy with the current shape?